packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm

```diff
@@ -1656,6 +1666,146 @@ - (void)keyUp:(NSEvent *)event {
   }
 }
 
+
+#pragma mark - Mouse Events
+
+- (void)sendMouseEvent:(BOOL)isMouseOver {
```

Comment:
I think I slightly prefer `emitMouseEvent`, or `handleMouseEvent` to match `handleKeyboardEvent`